utils/polus-imagej-util/classes/populate.py

```diff
@@ -831,26 +1028,37 @@ class Plugin object to create the final cookiecutter json
     import jpype
     import os
     
-    print('Starting JVM and parsing ops help\n')
-    
-    # Populate ops by parsing the imagej operations help
-    populater = Populate()
-    
-    print('Building json templates\n')
-    
-    # Get the current working directory
-    cwd = Path(os.getcwd())
-    
-    # Save a directory for the cookietin json files
-    cookietin_path = cwd.joinpath('utils/polus-imagej-util/cookietin')
-    
-    # Build the json dictionary to be passed to the cookiecutter module 
-    populater.build_json('Benjamin Houghton', 'benjamin.houghton@axleinfo.com', 'bthoughton', '0.3.0', cookietin_path)
-    
-    print('Shutting down JVM\n')
+    try:
+        print('Starting JVM and parsing ops help\n')
+        
+        # Populate ops by parsing the imagej operations help
+        populater = Populate()
+        
+        print('Building json templates\n')
+        
+        # Get the current working directory
+        cwd = Path(os.getcwd())
+        
+        # Save a directory for the cookietin json files
+        cookietin_path = cwd.joinpath('utils/polus-imagej-util/cookietin')
+        
+        # Build the json dictionary to be passed to the cookiecutter module 
+        populater.build_json(
+            'Benjamin Houghton', 
+            'benjamin.houghton@axleinfo.com', 
+            'bthoughton', 
+            '0.5.0', 
+            cookietin_path)
+        
+        print('Shutting down JVM\n')
```

Comment:
This should probably be moved the `finally` block.

Reply:
This has been resolved